src/Data/Product/Properties/WithK.agda

```diff
@@ -8,33 +8,14 @@
 
 module Data.Product.Properties.WithK where
 
-open import Data.Bool.Base
 open import Data.Product
-open import Data.Product.Properties using (,-injectiveˡ)
 open import Function
-open import Relation.Binary using (Decidable)
 open import Relation.Binary.PropositionalEquality
-open import Relation.Nullary.Reflects
-open import Relation.Nullary using (Dec; _because_; yes; no)
-open import Relation.Nullary.Decidable using (map′)
 
 ------------------------------------------------------------------------
 -- Equality
 
-module _ {a b} {A : Set a} {B : Set b} where
-
-  ,-injective : ∀ {a c : A} {b d : B} → (a , b) ≡ (c , d) → a ≡ c × b ≡ d
```

Comment:
I would keep this proof here, I don't see it replicated in `Data.Product.Properties`

Reply:
Looks identical to the one in `Data.Product.Properties` to me

Reply:
Oh yes, doh. Didn't occur to me that it was duplicated. In that case we should re-export it anyway to preserve compatibility.

Reply:
sure